<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemWindowProfessor</class>
 <widget class="QMainWindow" name="SystemWindowProfessor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>946</width>
    <height>726</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_welcome">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>-10</y>
      <width>871</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Welcome</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>911</width>
      <height>561</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideLeft</enum>
    </property>
    <property name="usesScrollButtons">
     <bool>true</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Schedule</string>
     </attribute>
     <widget class="QTableWidget" name="tableSchedule">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>881</width>
        <height>491</height>
       </rect>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <property name="gridStyle">
       <enum>Qt::NoPen</enum>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>false</bool>
      </property>
      <property name="rowCount">
       <number>14</number>
      </property>
      <property name="columnCount">
       <number>7</number>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>100</number>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>37</number>
      </attribute>
      <row>
       <property name="text">
        <string>8</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>9</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>10</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>11</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>12</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>13</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>14</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>15</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>16</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>17</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>18</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>19</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>20</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>21</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Sunday</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Monday</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Tuesday</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Wednesday</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Thursday</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Friday</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Saturday</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="pbRefresh">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>My Courses</string>
     </attribute>
     <widget class="QTableWidget" name="tableMyCourses">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>831</width>
        <height>311</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Course Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Section Number</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Room</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Days</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Time</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Capacity</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Actual</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string/>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="pbAddSection">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>410</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Section</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbEditSection">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>410</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Edit Section</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbRemoveSection">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>410</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove Section</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Courses</string>
     </attribute>
     <widget class="QTableWidget" name="tableCourses">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>751</width>
        <height>311</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Section Number</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Room</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Days</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Time</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Capacity</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Actual</string>
       </property>
      </column>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>86</width>
        <height>47</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>List of Courses</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbCoursesList"/>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pbRequestCourse">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>420</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Request Course</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>946</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
